pantos/validatornode/business/transfers.py

```diff
@@ -908,11 +990,34 @@ def validate_transfer_task(self, internal_transfer_id: int,
                 'internal_transfer_id': internal_transfer_id,
                 'task_id': self.request.id
             }, exc_info=True)
-        retry_interval = config['tasks']['validate_transfer'][
-            'retry_interval_after_error_in_seconds']
+        retry_interval = _get_task_interval(self, after_error=True)
         raise self.retry(countdown=retry_interval, exc=error)
     if not validation_completed:
-        retry_interval = config['tasks']['validate_transfer'][
-            'retry_interval_in_seconds']
+        retry_interval = _get_task_interval(self)
         raise self.retry(countdown=retry_interval)
     return True
+
+
+def _get_task_interval(task, after_error: bool = False) -> int:
```

Comment:
Why does the celery task has no type?

Reply:
Because Celery also has no typing, there's no advantage in adding one here.